packages/react/src/components/createInlineOverlayComponent.tsx

```diff
@@ -55,6 +55,17 @@ export const createInlineOverlayComponent = <PropType, ElementType>(
     componentDidMount() {
       this.componentDidUpdate(this.props);
 
+      /**
+       * Mount the inner component when the
+       * overlay is about to open.
+       *
+       * For ion-popover, this is when `ionMount` is emitted.
+       * For other overlays, this is when `willPresent` is emitted.
+       */
+      this.ref.current?.addEventListener('ionMount', () => {
+        this.setState({ isOpen: true });
```

Comment:
Does this also need to call `onWillPresent` as in the `willPresent` listener below?

Reply:
No, when `willPresent` is emitted from the overlay, it will invoke that handler. The introduction of `ionMount` does not remove `willPresent` being emitted. 

The objective of `ionMount` is to mount the overlay contents sooner in the present lifecycle, so that transitions dependent on the bounding box are able to calculate dimensions correctly. All of the existing plumbing remains the same.

Reply:
Makes sense, thank you!